public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -1,13 +1,40 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import { css } from '@emotion/css';
 import { connect, ConnectedProps } from 'react-redux';
 import { getNavModel } from 'app/core/selectors/navModel';
 import Page from 'app/core/components/Page/Page';
 import { ServiceAccountProfile } from './ServiceAccountProfile';
 import { StoreState, ServiceAccountDTO, ApiKey } from 'app/types';
 import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
-import { deleteServiceAccountToken, loadServiceAccount, loadServiceAccountTokens } from './state/actions';
+import {
+  deleteServiceAccountToken,
+  loadServiceAccount,
+  loadServiceAccountTokens,
+  createServiceAccountToken,
+} from './state/actions';
 import { ServiceAccountTokensTable } from './ServiceAccountTokensTable';
-import { getTimeZone, NavModel } from '@grafana/data';
+import { getTimeZone, GrafanaTheme2, NavModel, OrgRole } from '@grafana/data';
+import {
+  Button,
+  ClipboardButton,
+  DatePickerWithInput,
+  Field,
+  FieldSet,
+  HorizontalGroup,
+  Icon,
+  Input,
+  Label,
+  Modal,
+  RadioButtonGroup,
+  useStyles2,
+  VerticalGroup,
+} from '@grafana/ui';
+import { getModalStyles } from '@grafana/ui/src/components/Modal/getModalStyles';
```

Comment:
If you need to import smth from package, like grafana/ui, it'd be exported in the index file of that package. Direct imports like these might break during build.

Reply:
@alexanderzobnin @Clarity-89 - in core it's ok to import directlry prom package/src - we do it a lot:)

Reply:
Ah ok, I remember it was causing some issues but I guess it has been fixed since.